InternTrack.Core.Api.Infrastructure.Provision/InternTrack - Backup.Core.Api.Infrastructure.Provision.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is this file needed.

Reply:
InternTrack.Core.Api.Infrastructure.Provision/InternTrack - Backup.Core.Api.Infrastructure.Provision.csproj

InternTrack - Backup???

Reply:
Sdk="Microsoft.NET.Sdk was needed. I dont know why exactly but provision layer would'nt reload after i deleted it.

Reply:
I don't know where it went. What should i do?

Reply:
Do you see the odd naming of this file? Maybe a git thing.

Yes the line you removed is typically needed in the .csproj file.